### PR TITLE
Updated the ozone driver to use variables instead of hardcoded values

### PR DESCRIPTION
**Pull request by CidVonHighwind**
This will be needed to be able to make the menu scalable.

### CODE REVIEW

File: menu/drivers/ozone/ozone.c [hunk 1]
```diff
@@ -986,11 +986,11 @@ static void ozone_draw_header(ozone_handle_t *ozone, video_frame_info_t *video_i
       ozone_draw_icon(video_info, 60, 60, ozone->textures[OZONE_TEXTURE_DISCORD_OWN_AVATAR], 47, 14, video_info->width, video_info->height, 0, 1, ozone->theme->entries_icon);
    else
 #endif
-      ozone_draw_icon(video_info, 60, 60, ozone->textures[OZONE_TEXTURE_RETROARCH], 47, 14, video_info->width, video_info->height, 0, 1, ozone->theme->entries_icon);
+      ozone_draw_icon(video_info, 60, 60, ozone->textures[OZONE_TEXTURE_RETROARCH], 47, (HEADER_HEIGHT - 60) / 2, video_info->width, video_info->height, 0, 1, ozone->theme->entries_icon);
    menu_display_blend_end(video_info);
 
    /* Battery */
-   if (video_info->battery_level_enable)
+   if (video_info->battery_level_enable || true)
```
Comment (natinusala) [inline menu/drivers/ozone/ozone.c]: Debug leftover

File: menu/drivers/ozone/ozone.h [hunk 2]
```diff
@@ -37,7 +37,12 @@ typedef struct ozone_handle ozone_handle_t;
 #define ANIMATION_CURSOR_DURATION 8
 #define ANIMATION_CURSOR_PULSE 30
 
-#define ENTRIES_START_Y 127
+#define HEADER_HEIGHT 87
+#define FOOTER_HEIGHT 78
+#define SIDEBAR_WIDTH 408
+#define ENTRIE_PADDING 67
```
Comment (natinusala) [inline menu/drivers/ozone/ozone.h]: Typo

**Comment by natinusala** [comment]
I will try this ASAP

**Comment by natinusala** [comment]
I noticed a few things:
- `ok_height` and `back_height` are now unused and can be removed
- the scale factor should be a `float` (unimportant for now) and be declared on top of the function for C89 compliance
- the structure doesn't need the `ui_` prefix because, well, ozone is already an ui (I'm being picky here)
- `/* TODO Fluid sidebar width ? */` can be removed in `ozone_sidebar.c`
- `SIDEBAR_ENTRY_ICON_WIDTH` is misleading - maybe break it into `SIDEBAR_ENTRY_ICON_SIZE + SIDEBAR_ENTRY_ICON_PADDING*2` ? With `SIDEBAR_ENTRY_ICON_PADDING` being at `15`

Other than that it's a really nice job, I can edit the values and the menu adapts itself just fine. I am happy with the result ! Fix those tiny things and it's going to be OK to merge on my end

**Comment by natinusala** [comment]
Looks good to me - I will fine tune the values once this is merged. Good work!

@twinaphex this can be merged

**Comment by natinusala** [comment]
Actually - @CidVonHighwind you should add yourself to the authors in the headers before merging

**Comment by CidVonHighwind** [comment]
Done!

**Pull request merged**